accord-core/src/test/java/accord/burn/BurnTest.java

```diff
@@ -217,21 +254,41 @@ static void burn(RandomSource random, TopologyFactory topologyFactory, List<Id>
                     .asLongSupplier(forked);
         };
 
-        Verifier verifier = createVerifier(keyCount);
-        SimulatedDelayedExecutorService globalExecutor = new SimulatedDelayedExecutorService(queue, null);
-
+        SimulatedDelayedExecutorService globalExecutor = new SimulatedDelayedExecutorService(queue, new ListAgent(1000L, failures::add, retryBootstrap, (i1, i2) -> {
+            throw new IllegalAccessError("Global executor should enver get a stale event");
+        }));
+        Int2ObjectHashMap<Verifier> validators = new Int2ObjectHashMap<>();
         Function<CommandStore, AsyncExecutor> executor = ignore -> globalExecutor;
 
         MessageListener listener = MessageListener.get();
 
-        Packet[] requests = toArray(generate(random, listener, executor, clients, nodes, keyCount, operations), Packet[]::new);
+        if (keyCount > ((long) HASH_RANGE_END - (long) HASH_RANGE_START + 1L))
+            throw new AssertionError(String.format("Attempted to create %d keys which is larger than the range (%d, %d]", keyCount, HASH_RANGE_START, HASH_RANGE_END));
```

Comment:
Why is that a bug? Multiple keys can map to a single hash. It should be fine to run with a hash range of [0..1) an 1000 keys if we want.

Reply:
ill get back to this comment later, you have a different comment this depends on so best to address things there.

https://github.com/apache/cassandra-accord/pull/65#discussion_r1442889538

Reply:
`BurnTest` now owns the hash function, so able to remove this check.